src/compiler/third-party-out-of-tree.md

```diff
@@ -0,0 +1,168 @@
+This is left to the discretion of compiler team members but should be discussed with the rest of
+the team, either through raising the question at the weekly triage meeting or asynchronously using
+[an approval decision](./proposals-and-stabilizations.md). If the crate is a product of a working
+group, there should already be agreement within the working group that an out-of-tree crate is
+suitable.
+
+When considering creating an out-of-tree crate, it is worth balancing how general the crate should
+be with the increased maintenance burden that this may bring if widely used.
+
+### Where should compiler crates live?
+Out-of-tree compiler crates should be hosted in the `rust-lang` organization - this simplifies
+integration with external infrastructure tooling and will inherit existing team permissions on
+GitHub. It should be made clear in any documentation that the compiler team and any appropriate
+working groups are responsible for the crate. It is not recommended to start with a prototype in
+another organization or personal repository.
+
+### Can existing out-of-tree crates from personal accounts or other organizations be transferred?
+Yes, this is encouraged. In order to do this, discuss this with the team and familiarize yourself
+with [the GitHub documentation for repository transfers][repo_transfers] and then arrange to perform
+the transfer. Once a transfer is complete, a redirect will exist in the original account or
+organization and this will conflict with the names of any new forks of the transferred repository -
+an email to GitHub is required to resolve this.
+
+[repo_transfers]: https://help.github.com/en/articles/transferring-a-repository
+
+### Who owns these crates?
+It is desired that a compiler team member or working group has loose ownership over a crate so
+that there are clear owners who are responsible for making sure that new versions are published and
+that pull requests are reviewed.
+
+### What should these crates be named?
+Crate naming will be discussed when new out-of-tree crates are proposed to the compiler team.
+
+Crate naming will differ on a case-by-case basis. Crates that are inherently tied to the
+compiler would benefit from a name that is prefixed with `rustc_`. This is an indicator of how
+stable the crate may be to prospective users. Other crates, which are more general-purpose, will
+have names that are disentangled from the compiler.
+
+### Are there any limitations on the review policy for out-of-tree crates?
+Generally, the working groups and team members that are primarily free to maintain the crate using
+whatever practices are best suited to their group, however, there are some limitations so that there
+is some uniformity across the compiler and out-of-tree crates:
+
+- Everyone with r+ on `rust-lang/rust` should be able to review and approve PRs.
+- Where possible, only active participants in the crate (or related working group) need be on the
+  highfive rotation for the crate.
```

Comment:
`highfive` hasn't handle the review rotation for some time now.
```suggestion
- Where possible, only active participants in the crate (or related working group) need be on the
  triagebot rotation for the crate.
```

Reply:
Fixed